src/docs/blueprint_download.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Eclipse Theia Blueprint Product
+---
+
+# Eclipse Theia Blueprint Product
+
+<img src="/theia-logo-blueprint.png" alt="Theia Logo Blue" style="max-width: 525px">
+
+## What is this?
+
+Eclipse Theia Blueprint product is a **template** to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform. Theia Blueprint assembles a selected subset of existing Theia features and extensions. We provide installers for Theia Blueprint to be downloaded (see below) as well as documentation on how to customize this template to build a product and installers for your own Theia-based product.
+
+## What is it not?
+
+Eclipse Theia Blueprint product is ***not*** **a production-ready product**. Therefore, it is also not a replacement for Visual Studio Code or any other IDE.
+
+## Downloads
+
+<span style="color:red">Warning: Currently Eclipse Theia Blueprint product is an **alpha** version. Please consider the current versions as experimental. </span>
+
+<table cellspacing="25">
+  <thead>
+    <tr>
+      <th>Windows</th>
+      <th>Linux</th>
+      <th>MacOS</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.0/windows/Theia-1.9.0.exe&r=1">Download 1.9.0</a></td>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.0/linux/Theia-1.9.0.AppImage&r=1">Download 1.9.0</a></td>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.0/macos/Theia-1.9.0.pkg&r=1">Download 1.9.0</a></td>
```

Comment:
I believe we should reference a constant url to download, referencing a version will become outdated pretty quickly and will require too much maintenance on the website.

Reply:
While we deploy all the latest versions to https://download.eclipse.org/theia/latest/ as well, the file names still contain the version number. So at least for now we don't have a static link to the last version. We will have to adapt the deployment scripts first.